src/coverlet.core/Instrumentation/Instrumenter.cs

```diff
@@ -61,6 +65,10 @@ private void InstrumentModule()
 
                     foreach (var method in type.Methods)
                     {
+                        var sourceFiles = method.DebugInformation.SequencePoints.Select(s => s.Document.Url).Distinct();
+                        if (_excludedFiles != null && sourceFiles.Any(_excludedFiles.Contains)) {
```

Comment:
Odd that the method could be declared in multiple files (I guess this is the case with `partial methods` in `partial classes`). I'm wondering if in this case, you'd only want to exclude the method if `sourceFiles.All(...)`.  @tonerdo you're obviously a lot more familiar with this - am I interpreting this correctly?

Reply:
They are part of the language: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/partial-method

If it isn't something that is supported in coverlet, then I guess we don't need to worry about it.

Reply:
OMG I've never even heard of `partial methods` until today. I guess we learn everyday. Taking my new found knowledge into consideration I'll still go with `.First(...)`, reason being that sequence points start at the body of the method and no matter how many partials of the same method you have, you can only have one implementation and the document that implementation is contained in is where all sequence points for that method will point to